app/models/user.rb

```diff
@@ -1,6 +1,6 @@
 class User < ApplicationRecord
-  has_and_belongs_to_many :servers
-  has_many :servers
+  has_and_belongs_to_many :servers, join_table: "users_servers"
+  has_many :answers
```

Comment:
Remove repetitive `servers`, I think this must have been a mistake and was supposed to be `answers`.